Sources/FluidAudioCLI/Utils/TextNormalizerOfficial.swift

```diff
@@ -0,0 +1,398 @@
+import Foundation
+import RegexBuilder
+
+/// Official Text Normalizer ported from HuggingFace OpenASR Leaderboard
```

Comment:
maybe use one or the other, officall vs othe other one seems weird and hacky imo